### PR TITLE
Demo tobe merged master

### PR DESCRIPTION
**Pull request by yogendraU**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by AlokDatta — APPROVED** [review]

File: case_creation.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+module "case-mgmt-api" {
+  source = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+
+  api_mgmt_name = local.api_mgmt_name
+  api_mgmt_rg   = local.api_mgmt_rg
+  revision      = "1"
+  service_url   = local.prl_api_url
+  product_id    = module.api-case-mgmt-product.product_id
+  name          = join("-", [var.product_name, "api"])
```
Comment (timja) [inline case_creation.tf:20]: much easier to read:
```suggestion
  name          = "${product_name}-api"
```

File: case_creation.tf [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  display_name  = "Case creation api"
+  path          = "prl-cos-api"
+  protocols     = ["http", "https"]
+  swagger_url   = "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/specs/case_creation.json"
+}
+
+data "template_file" "api_mgmt_policy_template" {
+  template = file(join("", [path.module, "/template/api-policy.xml"]))
```
Comment (timja) [inline case_creation.tf:28]:
```suggestion
  template = file("${path.module}/template/api-policy.xml")
```

File: case_document.tf [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+module "document-mgmt-api" {
+  source = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+
+  api_mgmt_name = local.api_mgmt_name
+  api_mgmt_rg   = local.api_mgmt_rg
+  revision      = "1"
+  service_url   = local.prl_api_url
+  product_id    = module.document-mgmt-product.product_id
+  name          = join("-", [var.document_product_name, "api"])
```
Comment (timja) [inline case_document.tf:20]:
```suggestion
  name          = "${var.document_product_name}-api"
```

File: case_document.tf [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  display_name  = "Case document api"
+  path          = "prl-document-api"
+  protocols     = ["http", "https"]
```
Comment (timja) [inline case_document.tf:23]: why http?

File: case_get_document.tf [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+module "case-document-mgmt-api" {
+  source = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+
+  api_mgmt_name = local.api_mgmt_name
+  api_mgmt_rg   = local.api_mgmt_rg
+  revision      = "1"
+  service_url   = local.prl_api_url
+  product_id    = module.case-document-mgmt-product.product_id
+  name          = join("-", [var.document_product_get_name, "api"])
```
Comment (timja) [inline case_get_document.tf:21]:
```suggestion
  name          = "${var.document_product_get_name}-api"
```

File: case_search.tf [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+module "case-search-mgmt-api" {
+  source = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+
+  api_mgmt_name = local.api_mgmt_name
+  api_mgmt_rg   = local.api_mgmt_rg
+  revision      = "1"
+  service_url   = local.prl_api_url
+  product_id    = module.api-case-search-mgmt-product.product_id
+  name          = join("-", [var.case_search_name, "api"])
```
Comment (timja) [inline case_search.tf:22]:
```suggestion
  name          = "${var.case_search_name}-api"
```

File: case_search.tf [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  display_name  = "Search cases for family api"
+  path          = "family-api"
+  protocols     = ["http", "https"]
+  swagger_url   = "https://raw.githubusercontent.com/hmcts/reform-api-docs/master/docs/specs/cafcass-case-search.json"
+
+}
+
+data "template_file" "case_mgmt_policy_template" {
+  template = file(join("", [path.module, "/template/api-policy.xml"]))
```
Comment (timja) [inline case_search.tf:31]:
```suggestion
  template = file("${path.module}/template/api-policy.xml")
```

**Review by timja — COMMENTED** [review]
makes the code easier to read

File: case_document.tf (hunk 4 above)
Comment (yogendraU) [reply case_document.tf]: removed http as protocols

**Pull request merged**